e2e/whisper/whisper_mailbox_test.go

```diff
@@ -248,23 +334,26 @@ func (s *WhisperMailboxSuite) startBackend() (*api.StatusBackend, func()) {
 		s.NoError(err)
 		<-backendStopped
 		s.False(backend.IsNodeRunning())
+		os.RemoveAll(datadir)
 	}
 
 }
+
+//Start mailbox node
 func (s *WhisperMailboxSuite) startMailboxBackend() (*api.StatusBackend, func()) {
-	//Start mailbox node
 	mailboxBackend := api.NewStatusBackend()
 	mailboxConfig, err := e2e.MakeTestNodeConfig(GetNetworkID())
 	s.Require().NoError(err)
+	datadir := "../../.ethereumtest/mailbox/mailbox/"
 
 	mailboxConfig.LightEthConfig.Enabled = false
 	mailboxConfig.WhisperConfig.Enabled = true
-	mailboxConfig.KeyStoreDir = "../../.ethereumtest/mailbox/"
+	mailboxConfig.KeyStoreDir = "../../.ethereumtest/mailbox/mailbox"
```

Comment:
Maybe this path could look like this `../../.ethereumtest/mailbox/mailserver`? `mailbox/mailbox` looks like a bug with too many nested directories.

Reply:
Good point. Thanks!